openomni/packet.py

```diff
@@ -66,6 +66,10 @@ def __init__(self, data=""):
             segment_len = min(Packet.MAX_BODY_SEGMENT_LEN,self.body_len,len(data)-14)
             self.body = data[13:(13+segment_len)]
             self.crc = ord(data[13+segment_len])
+            if (self.message_type[0].encode('hex') == '1d'):
+                self.minutes_active = ((ord(data[18]) & 0x0f) << 6) + (ord(data[19]) >> 2)
+                self.reservoir_level = round((float((((ord(data[19]) & 0x03) << 6) + (ord(data[20]) >> 2)))*50)/255, 0)
```

Comment:
Not sure I follow (keeping in mind I'm a relative python newb) because packet.py is currently testing for specific packet types and splitting data out into ex: ID, body, len, etc. and returning the string to print.  If you can give me a nudge in the direction you're thinking, I'll work on it.